hpcgap/lib/vecmat.gd

```diff
@@ -431,6 +431,34 @@ DeclareOperation( "ImmutableMatrix",[IsObject,IsMatrix]);
 
 #############################################################################
 ##
+#F  ImmutableVector( <field>, <vector>[, <change>] ) . convert into "best" representation
+##
+##  <#GAPDoc Label="ImmutableVector">
+##  <ManSection>
+##  <Oper Name="ImmutableVector" Arg='field, vector[, change]'/>
+##
+##  <Description>
+##  returns an immutable vector equal to <A>vector</A> which is in the optimal
+##  (concerning space and runtime) representation for vectors defined over
+##  <A>field</A>. This means that vectors obtained by several calls of
+##  <Ref Oper="ImmutableVector"/> for the same <A>field</A> are compatible
+##  for fast arithmetic without need for field conversion.
+##  <P/>
+##  The input vector <A>vector</A> might change the representation,
```

Comment:
Perhaps -- I would certainly expect both to print something. But beyond that, I'd wonder why there is a special function `PrintVector`, instead of a method for `Print` that prints vectors, and would assume that there either must be some semantic difference, or some historic quirks...

Anyway, I actually do not want to argue to argue too much for the merits of this function -- as already explained, I only picked the name and semantics to match the existing function `ImmutableMatrix`; if it wasn't for that, I'd not mind renaming this function, changing its semantics, etc. etc. more or less at will... As it is, though, these "bigger" changes would seem to be best suited for the bigger `MatrixObj` project.